src/pages/LandingPage.js

```diff
@@ -67,20 +64,29 @@ const makeCard = (link, title, body) => h(Clickable, {
   makeRightArrowWithBackgroundIcon()
 ])
 
+const getSiteSpecificHyperlinks = () => Utils.cond(
+  [isTerra(), () => [
+    makeDocLink('https://support.terra.bio/hc/en-us', 'Find how-to\'s, documentation, video tutorials, and discussion forums'),
+    makeDocLink('https://support.terra.bio/hc/en-us/articles/360033416672', 'Learn more about the Terra platform and our co-branded sites')
+  ]],
+  [isFirecloud(), () => [
+    makeDocLink('https://support.terra.bio/hc/en-us', 'Find how-to\'s, documentation, video tutorials, and discussion forums'),
+    makeDocLink('https://support.terra.bio/hc/en-us/articles/360022694271', 'Already a FireCloud user? Learn what\'s new.'),
+    makeDocLink('https://support.terra.bio/hc/en-us/articles/360033416912',
+      'Learn more about the Cancer Research Data Commons and other NCI Cloud Resources')
+  ]],
+  [isRareX(), () => [
+    makeDocLink('https://rare-x.org/DataAnalysisPlatform-Documentation',
+      'Find RARE-X Data Analysis Platform documentation, tutorials and Jupyter notebook examples'),
+    makeDocLink('https://rare-x.org/researchers/', 'Learn more about the RARE-X Data Analysis Platform'),
+    makeDocLink('https://support.terra.bio/hc/en-us', 'Find Terra how-to\'s, documentation, video tutorials, and discussion forums')
+  ]]
+)
```

Comment:
This is missing a default case. Currently, all themes show this link:
```js
makeDocLink('https://support.terra.bio/hc/en-us', 'Find how-to\'s, documentation, video tutorials, and discussion forums')
```

Reply:
Ugh - thanks for catching this! Fixed in [e45589e](https://github.com/DataBiosphere/terra-ui/pull/3212/commits/e45589e68730eb53fb6507e826980dfab38d4c1d).